diesel_migrations/migrations_internals/src/lib.rs

```diff
@@ -41,7 +41,7 @@
 //!
 //! Migrations can either be run with the CLI or embedded into the compiled application
 //! and executed with code, for example right after establishing a database connection.
-//! For more information, consult the [`embed_migrations!`](../macro.embed_migrations.html) macro.
+//! For more information, consult the [`embed_migrations!`](../../diesel::embed_migrations!) macro.
```

Comment:
That link seems to be wrong
```suggestion
//! For more information, consult the [`embed_migrations!`](crate::embed_migrations!) macro.
```